packages/analytics/analytics-utilities/package.json

```diff
@@ -60,6 +60,7 @@
     "date-fns-tz": "^2.0.0"
   },
   "devDependencies": {
+    "@kong-ui-public/i18n": "workspace:^",
```

Comment:
This is more than a dev dependency, right?  I think this needs to be a `peerDependency` as well.  `analytics-metric-provider` has an example; if I recall correctly, you need to add it to `peerDependencies`, `devDependencies`, and make some changes to `vite.config`.

However, see other comments; you might not need i18n in `timeframes`.

Reply:
ty for the heads up on that, fixing.

Reply:
Installed with `add --save-peer`. That makes it pop up in both devDependencies` and `peerDependencies` - which seems to be the expected behaviour, judging by other instances in this repo 👍